### PR TITLE
Reset events

### PR DESCRIPTION
**Pull request by nebolax**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Comment by nebolax** [comment]
A couple of tests failed, but they are are about assets / balances and are not related.

File: rotkehlchen/db/upgrades/v35_v36.py [hunk 1]
```diff
@@ -616,6 +587,28 @@ def _fix_eth2_pnl_genesis(write_cursor: 'DBCursor') -> None:
         )
 
 
+def _reset_decoded_events(write_cursor: 'DBCursor') -> None:
+    write_cursor.execute('SELECT tx_hash from evm_transactions')
```
Comment (yabirgb) [inline rotkehlchen/db/upgrades/v35_v36.py:591]: when writting such a functions always add a docstring explaining what it does

File: rotkehlchen/db/upgrades/v35_v36.py [hunk 2]
```diff
@@ -616,6 +587,28 @@ def _fix_eth2_pnl_genesis(write_cursor: 'DBCursor') -> None:
...
+    tx_hashes = [x[0] for x in write_cursor]
+    write_cursor.execute(
+        'SELECT parent_identifier FROM history_events_mappings WHERE name=? AND value=?',
+        (HISTORY_MAPPING_KEY_STATE, HISTORY_MAPPING_STATE_CUSTOMIZED),
+    )
+    customized_event_ids = [x[0] for x in write_cursor]
```
Comment (yabirgb) [inline rotkehlchen/db/upgrades/v35_v36.py:597]: in line 591 you are fetching all the tx hashes right? and in line 593 you are getting the customized ones and then you say 'Delete from history events where tx_hash in all and not in customized'. Can't you just do
```
ids_to_delete = list(set(all tx_hash) - set(customized))
```

and say

delete if in `ids_to_delete`. Also I'm not 100% sure but I believe doing a subquery is faster than checking if `event_identifier IN (?,....,?)`

File: rotkehlchen/tests/db/test_db_upgrades.py [hunk 3]
```diff
@@ -1202,21 +1202,8 @@ def test_upgrade_db_35_to_36(user_data_dir):  # pylint: disable=unused-argument
...
-            elif i == 1:
-                assert entry[i] == 1
-            else:
-                if entry[i] == 0:
-                    assert tx_mappings_result[idx][i] == 'decoded'
-                elif entry[i] == 1:
-                    assert tx_mappings_result[idx][i] == 'customized'
-                else:
-                    raise AssertionError(f'Unexpected value {entry[i]} in evm_tx_mappings')
-    assert cursor.execute('SELECT COUNT(*) from history_events').fetchone()[0] == 562
```
Comment (yabirgb) [inline rotkehlchen/tests/db/test_db_upgrades.py:1219]: all the history events are evm_transactions? If there was a customized event there should be at least one that is not right?

File: rotkehlchen/db/upgrades/v35_v36.py (hunk 1 above)
Comment (yabirgb) [reply rotkehlchen/db/upgrades/v35_v36.py:591]: Can you also state that this needs to happen after `_upgrade_events_mappings` since is altering the schema

**Review by yabirgb — COMMENTED** [review]
Left some comments

File: rotkehlchen/db/upgrades/v35_v36.py (hunk 2 above)
Comment (LefterisJP) [reply rotkehlchen/db/upgrades/v35_v36.py:597]: This is copy paste from the develop code that is tested. I would not change it. This should have been in the docstring.

File: rotkehlchen/db/schema.py [hunk 4]
```diff
@@ -634,6 +635,7 @@
 );
 """
 
+# This table is used to store for each history event it's chain id and whether it is customized.
```
Comment (LefterisJP) [inline rotkehlchen/db/schema.py:638]:
```suggestion
# This table is used to store for each history event its chain id and whether it is customized.
```

**Review by LefterisJP — APPROVED** [review]
Should be fine. Can merge if tests are okay. But please remember the typo for next PRs. I have done such nitpicks in other PRs. I hope you remember to fix them all because I don't remember if you follow up or not.

**Comment by LefterisJP** [comment]
3 tests failed with timeouts from coingecko

**Pull request merged**